lib/locale/pl_PL/strings.php

```diff
@@ -4721,6 +4721,7 @@
 
 $_LANG['Google Maps'] = 'Mapy Google';
 $_LANG['Copy link'] = 'Kopiuj link';
+$_LANG['Copy selected links'] = 'Kopiuj linki zaznaczonych';
```

Comment:
A może "Kopiuj zaznaczone jako linki"?

Reply:
Chciałem, żeby był jak najkrótszy napis na przycisku. Poprawiłem.

Reply:
To może "Kopiuj jako linki"?

Reply:
Zrobione.